### DISCUSSION TITLE
Best way to recover after a "Speech recognition is not available now" error message

### DISCUSSION THREAD

**Discussion by db1981** [react-native-voice/voice#418] [General]
Hello everyone,
we are using the react-native-voice library for an iOS app. Every now and then when we launch the Voice.start('en-US') method we get a "Speech recognition is not available now" error message back, and then a "Speech recognition already started!" subsequent message.
From that moment on it seems that the Voice object is corrupted and won't work anymore. We need to kill the app and start it from scratch.

What would be the best course to recover from such an error? Calling a Voice.destroy()?

Thanks!

**Comment by safaiyeh** [comment]
This project has been deprecated and archived. Please migrate to [`expo-speech-recognition`](https://github.com/jamsch/expo-speech-recognition). Thank you! ❤️
